src/cc65/declare.c

```diff
@@ -771,17 +771,19 @@ static SymEntry* ParseStructDecl (const char* Name)
             */
             if (BitOffs > 0) {
                 if (FieldWidth <= 0 || (BitOffs + FieldWidth) > INT_BITS) {
+                    /* Bits needed to byte-align the next field. */
+                    unsigned PaddingBitWidth = -BitOffs % CHAR_BITS;
```

Comment:
This in MSVC generates [C2220](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2220?view=vs-2019) + [C4146](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-2-c4146?view=vs-2019) and causes building to fail.

Reply:
It's right and intended in this case. 

I can blindly follow https://stackoverflow.com/a/25447795 and do
```
#pragma warning(suppress: 4164)
```
but I can't test it.

Something silly like `(CHAR_BITS - BitOffs) % CHAR_BITS` might fix this warning, but I don't know if it causes another.

Reply:
Sent #1079 for fix. Sorry about that.

Reply:
Merged https://github.com/cc65/cc65/pull/1079.